content/local/store.go

```diff
@@ -546,11 +546,11 @@ func (s *store) writer(ctx context.Context, ref string, total int64, expected di
 			return nil, err
 		}
 
-		if writeTimestampFile(filepath.Join(path, "startedat"), startedAt); err != nil {
+		if err := writeTimestampFile(filepath.Join(path, "startedat"), startedAt); err != nil {
 			return nil, err
 		}
 
-		if writeTimestampFile(filepath.Join(path, "updatedat"), startedAt); err != nil {
+		if err := writeTimestampFile(filepath.Join(path, "updatedat"), startedAt); err != nil {
```

Comment:
These were fixed as part of https://github.com/containerd/containerd/pull/3160 in master, but that was a bigger change (https://github.com/containerd/containerd/pull/3191 could've been used, but it wasn't merged in favour of the other one 😂)